src/swarm-lang/Swarm/Language/Syntax/Loc.hs

```diff
@@ -66,10 +66,10 @@ srcLocEndsBefore :: SrcLoc -> SrcLoc -> Bool
 srcLocEndsBefore (SrcLoc _ a) (SrcLoc _ b) = a <= b
 srcLocEndsBefore _ _ = False
 
--- | A variable with associated source location, used for variable
---   binding sites. (Variable occurrences are a bare TVar which gets
---   wrapped in a Syntax node, so we don't need LocVar for those.)
-data Located v = LV {lvSrcLoc :: SrcLoc, lvVar :: v}
+-- | A type with associated source location.
+data Located t = LV {lvSrcLoc :: SrcLoc, inner :: t}
```

Comment:
I thought about the `inner` name for awhile and thought it suggested the wrapping nature of `Located` but happy for other suggestions